examples/simplereader.c

```diff
@@ -178,7 +178,7 @@ int main(int argc, char **argv)
 
     if (err) {
         fprintf(stderr, "CBOR parsing failure at offset %ld: %s\n",
-                it.ptr - buf, cbor_error_string(err));
+                it.source.ptr - buf, cbor_error_string(err));
```

Comment:
This is correct, but can you use `cbor_value_get_next_byte(&it)` instead?

Reply:
Good idea, I will adapt this PR. I'll need a bit of time...